lib/pf/services/manager/haproxy.pm

```diff
@@ -58,6 +70,17 @@ sub generateConfig {
     $tags{'http'} = '';
     $tags{'mysql_backend'} = '';
     $tags{'var_dir'} = $var_dir;
+    my $bind_process = '';
+    if ($self->_number_cpus > 1) {
```

Comment:
Strange condition since there is no else.
What happen in the (improbable) case that there is only one CPU ?